src/components/Application.tsx

```diff
@@ -401,10 +402,15 @@ export class Application extends React.Component<Props, State> {
       jobs: this.state.jobs.$filter(j => j.id !== id),
     })
     if (this.state.route.jobIds.includes(id)) {
+      const job = this.state.jobs.records.find(j => j.id === id)
```

Comment:
Can we move this variable declaration closer to where it's used?  It's only needed in the `forgetJob` call's `catch` block.